README.md

```diff
@@ -205,3 +205,60 @@ build information to be returned by `/ping.json`. e.g.:
 These are required in order to submit user feedback to Zendesk.
 
 `ZENDESK_URL` defaults to `https://ministryofjustice.zendesk.com/api/v2`.
+
+## Smoke Tests
+
+Runs the happy path to book, confirm and cancel a visit.  If the app
+receives a an email in the format of
+`SMOKE_TEST_EMAIL_LOCAL_PART`@`SMOKE_TEST_EMAIL_DOMAIN`, it will not
+send that email to the prison, but will direct all mail the
+`SMOKE_TEST_EMAIL_HOST`.
+
+To start the smoke tests, set the environment variables and run
+`ruby ./smoke_test/smoke_test.rb`.
+
+The normal SMTP server details will need to be set and the app must be
+run in production mode for the smoke tests to work.
+
+##### Smoke Test Steps:
+  - [x] Starts the form
+  - [x] Fills in prisoner details page
+  - [x] Fills in visitor details page for one visitor
+  - [x] Selects three available days on the slot page
+  - [x] Submits the booking on the check your request page
+  - [x] Checks that the user received a booking receipt email
+  - [x] Checks that the prison received a booking request email
+  - [x] Visits the process booking page and confirms the booking
+  - [x] Checks that the prison received a copy of the confirmation email
+  - [x] Checks that the visitor received a confirmation email
+  - [x] Checks the status of the visit and cancels it
+  - [x] Checks that the prison receives a booking cancellation email
```

Comment:
Why checkboxes? As this all works now, it might be nice if this were an ordered list…